elyra/cli.py

```diff
@@ -154,42 +95,23 @@ def submit(pipeline_path, name):
     """TODO: Description"""
     click.echo()
 
-    click.echo(Fore.CYAN + "────────────────────────────────────────────────────────────────" + Style.RESET_ALL)
-    click.echo(Fore.CYAN + "  Elyra Pipeline Submission" + Style.RESET_ALL)
-    click.echo(Fore.CYAN + "────────────────────────────────────────────────────────────────" + Style.RESET_ALL)
-    click.echo()
-
-    click.echo(Fore.CYAN + "❯ Version" + Style.RESET_ALL)
-    click.echo("  elyra " + __version__)
-    click.echo()
+    print_banner("Elyra Pipeline Submission")
+    print_version()
     
-    pipeline = prepare_pipeline(pipeline_path, name=name, runtime='kfp', runtime_config='fun')
-
-    click.echo(Fore.CYAN + "❯ Info" + Style.RESET_ALL)
-    click.echo("  name: " + pipeline.name)
-    click.echo()
+    pipeline_definition = prepare_pipeline(pipeline_path, name=name, runtime='kfp', runtime_config='fun')
+    pipeline = PipelineParser().parse(pipeline_definition)
 
-    click.echo(Fore.CYAN + "❯ Runtime" + Style.RESET_ALL)
-    click.echo("  type: " + pipeline.runtime)
-    click.echo("  config: " + pipeline.runtime_config)
-    click.echo()
+    print_info("Info", [("name", pipeline.name)])
+    print_info("Runtime", [("type", pipeline.runtime), ("config", pipeline.runtime_config)])
     
     with yaspin(text="Submitting Pipeline..."):
         os.environ["ELYRA_METADATA_PATH"] = os.path.join(os.path.expanduser("~"), ".elyra")
```

Comment:
I don't think this should be here.  If users which to alter where metadata instances are stored, this should be set in their environment already and not something we impose in the tool.

Reply:
No yea I agree, I just have my config sitting there right now (I also have the runtime config stuff hardcoded here too)